lib/config/common.ts

```diff
@@ -73,6 +73,10 @@ export interface RenovateAdminConfig {
   baseDir?: string;
   cacheDir?: string;
   configWarningReuseIssue?: boolean;
+
+  dockerImagePrefix?: string;
+  dockerUser?: string;
```

Comment:
Is this necessary for the PR?

Reply:
No, but was missing anyway. I don't want to add a pr for just that simple type.

There are a lot more properties missing but becuase of extending `Record` somewhere it will mostly work. We should make that types more strict.